src/main/scala/com/dwolla/cloudflare/domain/dto/accounts/AccountDTO.scala

```diff
@@ -0,0 +1,7 @@
+package com.dwolla.cloudflare.domain.dto.accounts
+
+private[cloudflare] case class AccountDTO (
```

Comment:
I think it would be useful to publish these DTOs as a separate artifact. Some users may not want to use the HTTP client we've embedded in this library, but having the DTOs available for parsing and outputting JSON that works with Cloudflare's API would still be useful.

Reply:
Done.